tests/dnfserver/container/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+echo "I'm alive!" > /certificates/index.html
+chown $myuser:nginx /certificates
+chmod 770 /certificates
+chown -R nginx:nginx "/certificates"
+
+# put certs in place
+mkdir "/etc/pki/nginx"
+chown -R nginx:nginx "/etc/pki/nginx"
+chmod 700 "/etc/pki/nginx"
+
+cp /certificates/localhost.crt "/etc/pki/nginx/server.crt"
+cp /certificates/localhost.key "/etc/pki/nginx/server.key"
+cp /certificates/myCA.crt "/etc/pki/myCA.crt"
+
+# create yum repo
+pip3 install createrepo_c
```

Comment:
it's there so that the lock file can be regenerated if needed from the dnfserver.

Reply:
> it's there so that the lock file can be regenerated if needed from the dnfserver.

What use case would that be useful for in context of integration tests?